python/lsst/ip/isr/isrTask.py

```diff
@@ -172,7 +173,7 @@ class IsrTaskConfig(pexConfig.Config):
              "or number of spline knots if overscan fit type is a spline."),
         default=1,
     )
-    overscanRej = pexConfig.Field(
+    overscanNumSigmaClip = pexConfig.Field(
```

Comment:
I think this change and the addition of `MEANCLIP` should be in their own commits, not hidden in a "modernisation" commit.

Reply:
I'll restructure the commits.